Update localizing-stores.md (#1139)
bigcommerce/dev-docs: docs/stencil-docs/localization/localizing-stores.md

### Multiple language strings by BigCommerce

> Currently, you cannot edit strings such as "This promotion cannot be applied with the selected currency" in BigCommerce. The language settings for these strings are not defined by the shopper's browser language settings. Rather, you select your storefront's display language by using the default language setting found in the control panel's [Store Profile](https://support.bigcommerce.com/s/article/Store-Profile-Settings#locale). This language setting determines the language of your storefront content.

Reviewer: ```suggestion
Currently, you cannot edit strings such as "This promotion cannot be applied with the selected currency" in BigCommerce. The language settings for these strings are not defined by the shopper's browser language settings. Rather, you select your storefront's display language using the default language setting found in the control panel's [Store Profile](https://support.bigcommerce.com/s/article/Store-Profile-Settings#locale). This language setting determines the language of your storefront content.

```

> Some of the strings displayed on your storefront are not currently editable in BigCommerce. These are strings such as "This promotion cannot be applied with the selected currency."  The language setting for these strings are not defined by the shopper's browser language settings. Rather, you should select the language displayed on your storefront by using the default language setting found in the control panel's [Store Profile](https://support.bigcommerce.com/s/article/Store-Profile-Settings#locale) area. This language setting determines the language of your storefront content.

Reviewer 2: ```suggestion
Strings such as "This promotion cannot be applied with the selected currency"  are not currently editable in BigCommerce. The language settings for these strings are not defined by the shopper's browser language settings. Rather, you select your storefront's display language by using the default language setting found in the control panel's [Store Profile](https://support.bigcommerce.com/s/article/Store-Profile-Settings#locale). This language setting determines the language of your storefront content.

```

I made some changes to make the text more concise but feel free to ignore it if it changes the meaning.

</div>
</div>

## Localizing your storefront content

> Localizing the control panel and options outside your theme and are accessible through the BigCommerce control panel. The [Default Language setting, found in the control panel's Store Profile area](https://support.bigcommerce.com/s/article/Store-Profile-Settings#locale), allows localization of the following items:

Reviewer 2: ```suggestion
The Default Language setting, found in the control panel's [Store Profile](https://login.bigcommerce.com/deep-links/settings/store-profile), allows localization of the following items:

```
I found the first sentence confusing. Can it be removed? I've tried to rewrite it but could not come up with anything.

### Managing transactional emails

> To fully localize a store into a language other than English, disable the sending of BigCommerce native emails. To disable native emails, go to **Storefront** › [**Email Templates**](http://login.bigcommerce.com/deep-links//manage/storefront-manager/email-templates). Uncheck each email type that you prefer to customize. If you would like to localize transactional emails, see our Partner Apps, such as [CM Commerce](https://www.bigcommerce.com/apps/cm-commerce/), for more details.

Reviewer 2: ```suggestion
To fully localize a store into a language other than English, disable the sending of BigCommerce native emails. To disable native emails, go to **Storefront** › [**Email Templates**](http://login.bigcommerce.com/deep-links/manage/storefront-manager/email-templates). Uncheck each email type that you prefer to customize. If you would like to localize transactional emails, see our Partner Apps, such as [CM Commerce](https://www.bigcommerce.com/apps/cm-commerce/), for more details.

```
